src/queries.ts

```diff
@@ -1,18 +1,17 @@
-import { gql } from "graphql.macro";
+import { graphql } from "./common/graphql/types";
 
-const COUNT_FRAGMENT = gql`
+graphql(`
   fragment Count on Characters {
     info {
       count
     }
   }
-`;
+`);
 
-export const GET_CHARACTER = gql`
-  ${COUNT_FRAGMENT}
```

Comment:
We can't inline fragments because it will break TypeScript inference, so instead, all fragments are defined "globally".